scripts-dev/complement.sh

```diff
@@ -65,4 +65,4 @@ if [[ -n "$1" ]]; then
 fi
 
 # Run the tests!
-go test -v -tags synapse_blacklist,msc2946,msc3083 -count=1 $EXTRA_COMPLEMENT_ARGS ./tests
+go test -v -tags synapse_blacklist,msc2946,msc3083,msc2716 -count=1 $EXTRA_COMPLEMENT_ARGS ./tests
```

Comment:
The last CI failures are the Complement tests (and retries).

Locally, they pass just fine so it's difficult to tell what's going wrong.

Reply:
@erikjohnston Oh, I had no idea [CI tried to match the same branch in the Complement repo](https://github.com/matrix-org/synapse/blob/a5cd05beeeac80df0352bd50c2ad2e017664665c/.github/workflows/tests.yml#L314-L319). Merged the latest `master` to my Complement branch, refactored, and skipped the federation tests ✅ 

CI is now passing here! ✅